### DISCUSSION TITLE
[Kavita] Ability to disable Tabs on Overview in Manga Overview

### DISCUSSION THREAD

**Discussion by ghost** [Kareadita/Kavita#2646] [Ideas]
### Idea Description

I would guess that many Users read Mangas by Chapternumber and not Storyline or Volumes, therefore it would be nice if you could remove the not needed Tabs from the Manga Overview.
Either global as Library / Serversetting or per User.

### Duration of Using Kavita

in Total 3+ Months

### Idea Category

User Experience

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
Storyline is exactly how I consume Manga btw.

**Reply by ghost** [reply]
None of my friends want to read ch 1, ch 52, ch 153, ch 7 ... (as an extreme example), we always read 1-99 in order.
All of us find it kinda annoying that Storyline is the first tab and not chapters. It would be very helpful to remove those or reorder them.

**Reply by majora2007** [reply]
But that is what the storyline tab does. It presents volumes (if you have them) then chapters, in order. 

You just hit the continue button to resume the reader or read to start at the first chapter/volume. It's literally created to mimic reading order.

For manga, I'm not understanding why there is a benefit to removing it. I can understand for comics where issues are generally collected separately to TPBs, but manga can have both and chapters are collected into volumes. As long as you read the chapters in order, it shouldn't matter.

**Reply by majora2007** [reply]
Here is an example of what it looks for me. Maybe a visual from your perspective will help me understand your case. 
![image](https://github.com/Kareadita/Kavita/assets/735851/889095bf-9c13-4060-b3f3-528e58f96ce0)

**Comment by ghost** [comment]
Ok i actually cant find that problem, i only got my friends complaining about it, will just tell em to shut up lol
